siliconcompiler/remote/client.py

```diff
@@ -395,25 +489,27 @@ def fetch_results(chip):
 
     # Unauthenticated jobs get a gzip archive, authenticated jobs get nested archives.
     # So we need to extract and delete those.
-    with tarfile.open(f'{job_hash}.tar.gz', 'r:gz') as tar:
-        tar.extractall()
+    with tarfile.open(f'{job_hash}{node}.tar.gz', 'r:gz') as tar:
+        tar.extractall(path=(node if node else ''))
```

Comment:
what does the tarball look like? does it have build/design/jobname/step/index or is it different?

Reply:
Once the upcoming `get_results/` changes are merged, it will have the results of the SC archive calls. So, it will contain the `('option', 'builddir')` top level with the subset of result files which get returned.